.github/workflows/python-linter.yml

```diff
@@ -0,0 +1,59 @@
+# Copyright 2020 The Magma Authors.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+name: python-linter
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - master
+      - 'v[0-9]+.[0-9]+.?[0-9]?*'
+      - ci/python-mypy-linter
+  pull_request:
+    branches:
+      - master
+      - 'v[0-9]+.[0-9]+.?[0-9]?*'
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  python_lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: dorny/paths-filter@v2
+        id: changes
+        with:
+          base: master
+          list-files: shell
+          filters: |
+            python:
+              - 'lte/gateway/python/**'
+              - 'orc8r/gateway/python/**'
+      - uses: ricardochaves/python-lint@v1.4.0
+        if: steps.changes.outputs.python == 'true'
+        with:
+          python-root-list: ${{ steps.changes.outputs.python_files }}
+          use-pylint: false
+          use-pycodestyle: false
+          use-flake8: false
+          use-black: false
+          use-mypy: true
+          use-isort: false
+          extra-pylint-options: ""
+          extra-pycodestyle-options: ""
+          extra-flake8-options: ""
+          extra-black-options: ""
+          # for packages without __init__.py the option `--namespace-packages` needs to be set.
+          # However, for packages with __init__.py this provokes import errors.
```

Comment:
Does this mean that the linter will have problems depending on whether or not there's an `__init__.py` near the changed file?